src/hardware/reelmagic/driver.cpp

```diff
@@ -1367,6 +1380,11 @@ void ReelMagic_Init(Section* sec)
 	assert(sec);
 	const auto section = static_cast<Section_prop*>(sec);
 
+	// Always register the driver's text messages, even if the card is
+	// disabled We cannot rely on the driver to register them because we
+	// only create the file if the user wants ReelMagic enabled.
+	FMPDRV::AddMessages();
```

Comment:
I would, in my opinion, be more sane to execute this from within `REELMAGIC_MaybeCreateFmpdrvExecutable` - to ensure that string order in the translation file is the same regardless of the configuration.

Reply:
Yes: only one place to add messages instead of two, that's even better, @FeralChild64 - thanks.